src/components/deploy-table.tag

```diff
@@ -0,0 +1,54 @@
+<DeployTable>
+
+  Service: <input name="service" onkeyup={ edit }
+            class="input { txt.length > 0 && 'is-success' } { list.length == 0 && 'is-danger'}"
+            placeholder="Try Media"/>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Build Time</th>
+        <th>Status</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr each="{ item,i in page }" key={ i }>
+        <td>{ item.name }</td>
+        <td>{ item.minutes } minutes(s)</td>
+        <td>{ item.success ? 'Success' : 'Fail' }</td>
+      </tr>
+      <tr if={ list.length == 0}>
+        <td colspan="3">No services found.</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <button if={ index > 0 } class="button" onclick={ prev }>Prev</button>
+  <button if={ index+1 < list.length/psize } class="button" onclick={ next }>Next</button>
+
+  <script type="es6">
+
+    this.index = 0
```

Comment:
Would it make sense to create a pagination processor mixin that consumes a data ref, query method ref, and page size that processes data proportionately?

ie;

```javascript
class PaginationProcessor {
  constructor(dataRef, updateRef, pageSize) {
    this.data = dataRef
    this.update = updateRef
    this.pageSize = pageSize
  }
  process(index) {
    this.update()
    return this.data.slice(index).slice(-1 * this.pageSize)
  }
}
```

Example: https://repl.it/repls/OffensiveLeafyNandu

Reply:
Took some creative freedom with this, let me know what you think